internal/api/handlers/operations_handler_test.go

```diff
@@ -454,6 +454,124 @@ func TestCancelOperation(t *testing.T) {
 	})
 }
 
+func TestGetOperation(t *testing.T) {
+	dates := []time.Time{
+		time.Time{}.AddDate(2020, 0, 0),
+		time.Time{}.AddDate(2020, 1, 0),
+		time.Time{}.AddDate(2020, 2, 0),
+	}
+
+	type tableTestInput struct {
+		Operation *operation.Operation
+	}
+	type tableTestOutput struct {
+		Operation *operation.Operation
+		Status    int
+		Err       error
+	}
+	type tableTests struct {
+		Description string
+		Input       tableTestInput
+		Output      tableTestOutput
+	}
+
+	test1 := tableTests{
+		Description: "succeed - returns operation",
```

Comment:
```suggestion
		Description: "returns operation when no error occur",
``` 
as we have discussed in the last weeks, we were trying to adopt some test name standards and we had agreed on something like `return <something> when <something happens>` WDYT?